definitions/infra-unix_host/definition.yml

```diff
@@ -0,0 +1,11 @@
+domain: INFRA
+type: UNIX_HOST
+synthesis:
+  name: hostname
+  identifier: hostname
```

Comment:
This attribute name might be a bit generic, I think it might make sense if it had some condition but wdyt @cianBuckley @srvaroa ?

Reply:
in the sample data we have for the Unix OHI; `hostname` is the most unique attribute available unfortunately.

Reply:
No problem, is there some condition that can be added to differentiate from another entity that might use that attribute though? for example, for redis telemetry they can use something like 'redis_' prefix.

Reply:
do you mean can we manually add a prefix to the definition to make it unique; or are you asking if there's another attribute in the payload that we can concatenate to make it unique?

the first option is open; I don't think it matters much what this looks like in our back end if it doesn't change the way our users see the data. `unix_host` vs. `hostname` isn't a huge difference IMO. 

the second option, I don't believe is viable without a change to the integration to update the way it reports data and manually create a unique identifier there.

Reply:
Oh, yes I meant the second one, using another payload attribute to ensure that we're able to differentiate. Thanks for confirming it.

Reply:
I don't think our team have any definitive answer on this - if you feel hostname is sufficient to be unique across the domain, entity type and account, then that's fantastic

Reply:
> Multiple definitions could work well if we had a metric attribute that could be used like that `unix:System`.

do we have an example of what this would look like? I can get a PR submitted to the Unix integration to add an `osVersion` field (or something like it); but I'd like to see a working example first so I can make sure I'm following correctly. Thanks!

Reply:
Sorry, you're right @IreneMLC This will literally match on anything with a `hostname` attribute - it seems too open. What would be ideal is something like (example): 
```
  conditions:
    - attribute: metricName
      prefix: etcd_
```
Without that, every data point that has `hostname` attribute populated will be organised as a unix host and i don't think we want that

@thezackm Can we see a sample of telemetry that you want to generate entities from?

Reply:
i'll close this thread and open a new one just to clear up the PR